integration-tests/src/test/java/com/datastax/oss/driver/core/metadata/SchemaIT.java

```diff
@@ -254,6 +254,20 @@ public void should_get_virtual_metadata() {
                   + "    unit text,\n"
                   + "    PRIMARY KEY (keyspace_name, table_name, task_id)\n"
                   + "); */",
+              // Cassandra 5.0
+              "/* VIRTUAL TABLE system_views.sstable_tasks (\n"
+                  + "    keyspace_name text,\n"
+                  + "    table_name text,\n"
+                  + "    task_id timeuuid,\n"
+                  + "    completion_ratio double,\n"
+                  + "    kind text,\n"
+                  + "    progress bigint,\n"
+                  + "    sstables int,\n"
+                  + "    target_directory text,\n"
+                  + "    total bigint,\n"
+                  + "    unit text,\n"
+                  + "    PRIMARY KEY (keyspace_name, table_name, task_id)\n"
+                  + "); */",
```

Comment:
Nit: the sequence of candidate sstable_tasks schemas goes DSE 6.8+/C* 4.0/C* 4.1.  According to that sequence this block should appear last, after C* 4.1.

Reply:
Fixed.